### PR TITLE
Removed deprecated keywords

### PR DESCRIPTION
**Pull request by dboitsov-spriker**
## PR Description

Removed all deprecated since v5.0 keywords:
- Run keyword unless -> Run keyword if (with condition reverse)
- Execute Javascript -> Evaluate Javascript (note, the arguments are different)

## Checklist
- [ v] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by OKrivtsova — APPROVED** [review]

**Comment by dboitsov-spriker** [comment]
As per Anton's request, removed Run Keyword If with IF-ELSE statements.
Run Keyword If is not deprecated, but generally, IF-ELSE is more recomended since Robot 4.0
https://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Run%20Keyword%20If
I assume codestyle recomendations should also be updated to avoid usage of both deprecated keywords and Run Keyword if after PR gets merged

File: resources/steps/pdp_steps.robot [hunk 1]
```diff
@@ -115,24 +122,31 @@ Yves: add product to wishlist:
...
     ${wishlistSelectorExists}=    Run Keyword And Return Status    Element Should Be Visible    ${pdp_wishlist_dropdown}
-    Run Keyword If    '${selectWishlist}'=='select'    Run keywords
-    ...    Wait Until Element Is Visible    xpath=//select[contains(@name,'wishlist-name')]
-    ...    AND    Wait Until Element Is Enabled    xpath=//select[contains(@name,'wishlist-name')]
-    ...    AND    Select From List By Value    xpath=//select[contains(@name,'wishlist-name')]    ${wishlistName}      
+    IF    '${selectWishlist}'=='select'
+        Run keywords
+            Wait Until Element Is Visible    xpath=//select[contains(@name,'wishlist-name')]
+                Wait Until Element Is Enabled    xpath=//select[contains(@name,'wishlist-name')]
+                Select From List By Value    xpath=//select[contains(@name,'wishlist-name')]    ${wishlistName}
```
Comment (OKrivtsova) [inline resources/steps/pdp_steps.robot:129]: fix tabs

File: resources/common/common.robot [hunk 2]
```diff
@@ -255,9 +255,13 @@ Try reloading page until element is/not appear:
     [Arguments]    ${element}    ${shouldBeDisplayed}    ${tries}=20    ${timeout}=1s
     FOR    ${index}    IN RANGE    0    ${tries}
         ${elementAppears}=    Run Keyword And Return Status    Page Should Contain Element    ${element}
-        Run Keyword If    '${shouldBeDisplayed}'=='true' and '${elementAppears}'=='False'    Run Keywords    Sleep    ${timeout}    AND    Reload
-        ...    ELSE    Run Keyword If    '${shouldBeDisplayed}'=='false' and '${elementAppears}'=='True'    Run Keywords    Sleep    ${timeout}    AND    Reload
-        ...    ELSE    Exit For Loop
+        IF    '${shouldBeDisplayed}'=='true' and '${elementAppears}'=='False'
+            Run Keywords    Sleep    ${timeout}    AND    Reload
+            ELSE IF    '${shouldBeDisplayed}'=='false' and '${elementAppears}'=='True'
```
[the remark this answers is not in the document]
Comment (dboitsov-spriker) [reply resources/common/common.robot:260]: In this case I believe tabulation is correct - it highlights nesting of keywords in IF-ELSE

File: resources/common/common_api.robot [hunk 3]
```diff
@@ -1272,8 +1272,8 @@ Find or create customer cart
         Save value to a variable:    [data][0][id]    cart_id
         ${hasCart}    Run Keyword and return status     Should not be empty    ${cart_id}
         Log    cart_id:${cart_id}
-        Run Keyword Unless    ${hasCart}    I send a POST request:    /carts    {"data": {"type": "carts","attributes": {"priceMode": "${gross_mode}","currency": "${currency_code_eur}","store": "${store_de}"}}}
-        Run Keyword Unless    ${hasCart}    Save value to a variable:    [data][id]    cart_id
+        IF    !${hasCart}    I send a POST request:    /carts    {"data": {"type": "carts","attributes": {"priceMode": "${gross_mode}","currency": "${currency_code_eur}","store": "${store_de}"}}}
+        IF    !${hasCart}    Save value to a variable:    [data][id]    cart_id
```
[the remark this answers is not in the document]
Comment (dboitsov-spriker) [reply resources/common/common_api.robot:1276]: fixed

File: resources/common/common.robot [hunk 4]
```diff
@@ -50,8 +50,8 @@ SuiteSetup
     Load Variables    ${env}
     New Browser    ${browser}    headless=${headless}    args=['--ignore-certificate-errors']
     Set Browser Timeout    ${browser_timeout}
-    Run Keyword if    '${headless}=true'    Create default Main Context
-    Run Keyword if    '${headless}=false'    Create default Main Context
+    IF    '${headless}=true'    Create default Main Context
```
Comment (AntonShopin) [inline resources/common/common.robot:53]: This comment is applicable to the whole PR: Robot can't parse inline conditions. 

The supported format is:
`IF    '${headless}=true'    Create default Main Context
     ELSE IF    '${headless}=true'    Create default Main Context
END`

File: resources/steps/pdp_steps.robot (hunk 1 above)
Comment (dboitsov-spriker) [reply resources/steps/pdp_steps.robot]: Fixed

**Pull request merged**